generator/src/js/helpers/normalizever.js

```diff
@@ -0,0 +1,14 @@
+import { coerce } from 'semver';
+
+export default (version) => {
+  const letterRE = /([abcdefghijlmnopqrstuvwxyz]+$)/;
+
+  // convert to string, just in case
+  version = String(version);
+
+  // get the prerelease tag
+  const prerelease = version.match(letterRE) !== null ? `-${version.match(letterRE)[0]}` : '';
```

Comment:
I think this may not do what we want.

For openssl it looks like [`1.1.0` is older than `1.1.0a`](https://www.openssl.org/news/changelog.html).

This conversion in `normalizever.js` would turn `1.1.0a` into `1.1.0-a`.

[`1.1.0-a` has a lower precedence than `1.1.0` in semver](https://semver.org/#spec-item-9) which isn't what we want.

Here's a solution to this
* Take the patch level and multiply it by `100`
* Take the single letter suffix for openssl and convert it to an ordinal number between 1 and 26
* Add the resulting ordinal number to the patch level

This would result in 
* `1.0.0e` becoming `1.0.5`
* `1.0.2b` becoming `1.0.202`
* `1.1.4k` becoming `1.1.411`

which will make all precedence comparisons work

Reply:
> Take the single letter suffix for openssl and convert it to an ordinal number between 1 and 26

note that the suffix doesn't have to be single-letter, during 0.9.8 time it got up to 2 letters